websitev2/docs/sdks/proxy-react.md

```diff
@@ -0,0 +1,92 @@
+---
+id: proxy-react
+title: React proxy SDK
+---
+
+> This library is meant to be used with the [unleash-proxy](https://github.com/Unleash/unleash-proxy). The proxy application layer will sit between your unleash instance and your client applications, and provides performance and security benefits. DO NOT TRY to connect this library directly to the unleash instance, as the datasets follow different formats because the proxy only returns evaluated toggle information.
+
+# Installation
+
+```
+npm install @unleash/unleash-proxy-react
```

Comment:
maybe late if this is already published, but would a better name for this module simply be:

`@unleash/proxy-client-react`

Reply:
Yes. I doubt anyone is using it yet. We could republish it?

Reply:
I think we should!

Reply:
Ok, I'll fix.